### ISSUE TITLE
Ngx-scanner not reading PDF_417 format 

### ISSUE THREAD

**Issue by ShevyakovaKate** [zxing-js/ngx-scanner#476]
**Describe the bug**
I am using Ngx-scanner version 3.3.0 in angular project (angular 11). When trying to scan PDF_417 barcode from driver licence card there is no result. Furthermore, scanner can recognize code if trim it from the card, but if I show the whole card, there is scan failure.

**To Reproduce**
Example  barcode:
![IMG_0441 (1)](https://user-images.githubusercontent.com/29502444/164462970-f6ace6a7-d2e3-4c2c-8973-478e23765e59.JPG)

**Desktop (please complete the following information):**
 - macOS
 - Browser Chrome
 - Version 100.0.4896.88

**Comment by werthdavid** [comment]
Sorry, I am not familiar with PDF_417. Does this work on ZXing-Java? You can also verify this online via https://zxing.org/w/decode.jspx

**Comment by ShevyakovaKate** [comment]
Yes, it works at this site. Could you please help to contact with someone, who is familiar with that type of barcodes?

**Comment by werthdavid** [comment]
Sorry I have no idea.
Maybe @odahcam ?

**Comment by ChasePilkington** [comment]
Following... I’m facing the same issue trying to read a PDF_417
